visualize-admin/visualization-tool
# Embed options

Adds an option in the embed popover to choose either minimal or standard view for the chart when it's embedded.

Technically, it adds a query parameter "embedOptions" that is then picked up by the view to hide some elements under the chart.

Fix #943 

How to test:

- Open on https://visualization-tool-git-feat-embed-options-ixt1.vercel.app/en/v/sO8qHplebSER?dataSource=Prod
- Click the embed options
- Choose either standard or minimal options.
- The page should refresh to show how the chart would be embedded (right now, on smaller screen, the refresh triggers a layout change, which makes the popover move a little, a possible solution could be to have an [alternate layout for published view](https://github.com/visualize-admin/visualization-tool/pull/948) but I am keeping the dicussion into the other PR)
- The codes should reflect the options and can be copy pasted into jsbin.com for example for testing. The resulting embedded chart should reflect the selected options
